student.rb

```diff
@@ -1,17 +1,18 @@
+require_relative 'highfiveable'
+
 class Student
+  include HighFiveable
+
   attr_reader :age, :phase
   attr_accessor :name
 
+
   def initialize(options = {})
     @phase = 1
     @age = options.fetch(:age, 0)
     @name = options.fetch(:name, "")
   end
 
-  def offer_high_five
-    "High five!"
-  end
-
   def set_phase(num)
     response = ""
     if num == @phase
```

Comment:
What is a common parent class between SeniorTeacher, ApprenticeTeacher and Student to which name, age and phase can be extracted?